Build/Makefile.rtos.RTThread

```diff
@@ -1,6 +1,7 @@
-C_SRCDIRS += $(HBIRD_SDK_RTOS)/src $(HBIRD_SDK_RTOS)/libcpu/risc-v/nuclei
+C_SRCDIRS += $(HBIRD_SDK_RTOS)/src $(HBIRD_SDK_RTOS)/components/finsh $(HBIRD_SDK_RTOS)/libcpu/risc-v/nuclei
```

Comment:
Please add a variable called RTTHREAD_MSH, if RTTHREAD_MSH=1, then enable msh compoment, otherwise msh code will not be included.